doc/trace_analysis.rst

```diff
@@ -0,0 +1,143 @@
+.. _analysis-page:
+
+*********************
+Kernel trace analysis
+*********************
+
+Introduction
+============
+
+LISA comes with a plethora of analysis functions based on `Ftrace
+<https://www.kernel.org/doc/Documentation/trace/ftrace.txt>`_ traces. Under the
+hood, we use :mod:`trappy` to convert the trace events into
+:class:`pandas.DataFrame` which are suited to handling large data sets.
+
+Trace
+=====
+
+Our :class:`~lisa.trace.Trace` takes an Ftrace ``trace.dat`` file as input (Systrace
+``trace.html`` are also accepted), and provides access to both the raw trace events,
+as well as some new dataframes built from analysing and aggregating trace events.
+
+You can create one like so::
+
+  trace = Trace("path/to/trace.dat", events=["sched_switch", "sched_wakeup"])
```

Comment:
For a simple example, we may even omit `events` since it parses all events by default.

Reply:
Does it? @derkling had some issues with that lately IIRC

Reply:
That's right, there are some outdated comments, but by looking at trappy's code, I can't figure out why it is not the case.

Reply:
Looks like it is related to scope == 'custom', which will only select a handful of events for some reasons, but not the one we usually care about (nothing from sched.py).